Warhammer-Fantasy-Roleplay-2nd-Edition/WHFRP2.html

```diff
@@ -576,7 +576,7 @@ <h4 class="sheet-center">
           </div>
           <div class="sheet-col-11 sheet-center sheet-pad-r-sm sheet-vert-middle">
             <select class="sheet-select-no-arrow-center" style="height: 17px" name="attr_Skill_Know_Lang">
-              <option value="@{Intelligence}" selected="selected"><span data-i18n="trained">Trained</span></option>
+              <option value="@{Intelligence}" selected="selected" data-i18n="trained">Trained></option>
```

Comment:
I needed to remove the span tag to make the translations works. I don't know if it is wanted, a bug or a missing feature :/